var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -35,6 +35,10 @@ class Vtk(CMakePackage):
     variant('ffmpeg', default=False, description='Build with FFMPEG support')
     variant('mpi', default=True, description='Enable MPI support')
 
+    # Haru causes trouble on Fedora and Ubuntu in v8.1.1
```

Comment:
You can go ahead and remove the libharu comments as we've already decided to just use the vendored libharu since it needs to be specially patched anyways.